scripts/generate_go_abi.sh

```
@@ -1,26 +1,134 @@
 
-    abigen --abi "$ABI_PATH/${CONTRACT}" --pkg $PACKAGE_NAME --out="$PACKAGE_DIR/${CONTRACT_NAME}.go"
+    # Create temporary bin file from the bytecode in JSON
+    TMP_BIN_FILE=$(mktemp)
+    jq -r '.bytecode' "$ARTIFACTS_PATH/$CONTRACT" >"$TMP_BIN_FILE"
+
+    # Create temporary ABI file from JSON
+    TMP_ABI_FILE=$(mktemp)
+    jq -r '.abi' "$ARTIFACTS_PATH/$CONTRACT" >"$TMP_ABI_FILE"
+
+    echo -e "Generating $PACKAGE_DIR/${WHITE}${CONTRACT_NAME}.go${NC}"
+    ABIGEN_CMD="abigen --abi "$TMP_ABI_FILE" --bin "$TMP_BIN_FILE" --pkg $PACKAGE_NAME --out="$PACKAGE_DIR/${CONTRACT_NAME}.go""
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)